src/components/Button.js

```diff
@@ -21,11 +25,40 @@ const Button = React.createClass({
 
   getDefaultProps () {
     return {
+      isActive: false,
       primaryColor: StyleConstants.Colors.PRIMARY,
       type: 'primary'
     };
   },
 
+  _renderButtonContent () {
+    const circleStyles = {
+      display: 'inline-block',
+      verticalAlign: 'middle',
+      borderRadius: '100%',
```

Comment:
try setting the border radius to the matching pixel height/width. It might fix the lopsidedness.